articles/quickstart/spa/react-beta/02-calling-an-api.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Calling an API
+description: This tutorial demonstrates how to make API calls for protected resources on your server.
+budicon: 546
+topics:
+  - quickstarts
+  - spa
+  - react
+  - apis
+github:
+  path: 02-Calling-an-API
+sample_download_required_data:
+  - client
+  - api
+contentType: tutorial
+useCase: quickstart
+---
+
+<!-- markdownlint-disable MD041 MD002 -->
+
+Most single-page apps use resources from data APIs. You may want to restrict access to those resources, so that only authenticated users with sufficient privileges can access them. Auth0 lets you manage access to these resources using [API Authorization](/api-auth).
+
+This tutorial shows you how to create a simple API using [Express](https://expressjs.com) that validates incoming JSON Web Tokens. You will then see how to call this API using an Access Token granted by the Auth0 authorization server.
+
+<%= include('../_includes/_calling_api_create_api') %>
+
+<%= include('../_includes/_calling_api_create_backend.md') %>
+
+Finally, modify `package.json` to add two new scripts `dev` and `server` that can be used to start the frontend and the backend API together:
+
+```json
+scripts": {
+  "start": "react-scripts start",
+  "build": "react-scripts build",
+  "test": "react-scripts test",
+  "eject": "react-scripts eject",
+  "now-build": "react-scripts build",
```

Comment:
should we add the now-build script? are we deploying to now as part of the QS?

Reply:
No we're not; will remove this